src/App.css

```diff
@@ -1,19 +1,105 @@
-.App {
-  text-align: center;
+/* App.css */
+.app-container {
+  display: flex;
+  flex-direction: column;
+  height: 100vh; /* height of 100vh ensures that the layout spans the full height */
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.task-list,
+.task-in-progress,
+.task-in-review,
+.task-completed {
+  flex: 1; /* Equal distribution of space */
+  box-sizing: border-box; /* Include padding and border in the width calculation */
+  width: calc(33.33% - 40px); /* 33.33% width with 20px margin on each side */
+  margin: 20px 20px;
+  border: 1px solid #ccc;
+  padding: 16px;
+  border-radius: 12px;
+  opacity: 80%;
+  font-size: 27px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.task-container {
   display: flex;
-  flex-direction: column;
-  align-items: center;
+  flex: 1; /* This allows the task components to take up the available space */
+}
+
+.header-2 {
```

Comment:
I think naming things with *-# is not exactly good. I would rather describe where the header is located or what kind of header it is instead of giving numbers